plugins/reach/lib/api/enums/KalturaCatalogItemLanguage.php

```diff
@@ -70,5 +70,31 @@ class KalturaCatalogItemLanguage extends KalturaStringEnum
 	const SL = KalturaLanguage::SL;
 	const TE = KalturaLanguage::TE;
 	const UG = KalturaLanguage::UG;
+	const EN_AU = KalturaLanguage::EN_AU;
```

Comment:
Some of these enums need to be added at:
api_v3/lib/types/enums/KalturaLanguage.php

KalturaLanguage::EN_AU
KalturaLanguage::BS
KalturaLanguage::FA_IR
KalturaLanguage::SK_SK
KalturaLanguage::ZH_CN
KalturaLanguage::ZH_HK

Reply:
Reviewed the list and added the missing ones.